RELEASES.md

```diff
@@ -1,12 +1,17 @@
 Unreleased
 ==========
-* Convert `Ascii` into an enum with a variant for every ASCII character.
+* Convert `Ascii` into an enum with a variant for every ASCII character and rename it to `AsciiChar`.
+* Replace `OwnedAsciiCast` with `IntoAsciiString`.
+* Replace `AsciiCast` with `AsAsciiStr` and `IntoAsciiChar`.
+* Replace *\_nocheck* with *\_unchecked*, and *from_bytes* with *from_ascii* in method names.
+* Return `std::error::Error` implementing types on error.
+* Implement `AsciiExt` without the `unstable` Cargo feature flag, which is  removed.
+* Require Rust 1.9 or later.
```

Comment:
Thats true. We can lave it as it is, there is hardly a "more important than the other" change in this list.